src/datatable/datatable.jl

```diff
@@ -149,44 +158,11 @@ function DataTable(column_eltypes::Vector, nrows::Integer)
     columns = Vector{Any}(p)
     cnames = gennames(p)
     for j in 1:p
-        columns[j] = NullableArray{column_eltypes[j]}(nrows)
+        columns[j] = Vector{column_eltypes[j]}(nrows)
     end
     return DataTable(columns, Index(cnames))
 end
 
-# Initialize from a Vector of Associatives (aka list of dicts)
```

Comment:
This seems like a strange constructor and so I'd opt to delete it outright rather than fix it, but I can add this back if desired